src/test/scala/com.snowplowanalytics.weather/providers/openweather/ExtractSpec.scala

```diff
@@ -18,6 +18,7 @@ import scala.io.Source
 
 // cats
 import cats.Id
+import cats.syntax.either._
```

Comment:
is this needed?

Reply:
This is to run these tests on 2.11 also, where Either is not right-biased. Alternatively I can change every `flatMap` call into `right.flatMap`.